Symfony/CS/Tokenizer/TokensAnalyzer.php

```diff
@@ -211,7 +211,84 @@ public function isArrayMultiLine($index)
     }
 
     /**
-     * Check if there is a lambda function under given index.
+     * Returns the attributes of the method under the given index.
+     *
+     * @param int $index Token index of the method (T_FUNCTION)
+     *
+     * @return array {
+     *     @var int|null $visibility T_PRIVATE, T_PROTECTED or T_PUBLIC
```

Comment:
Basically, array elements are not variables and their indexes are not denoted like that.

I can only assume that style of documentation is very specific to some kind of IDE, and is not generally accepted.